utils/config/tree_test.cpp

```diff
@@ -58,7 +58,7 @@ class int_wrapper {
     {
     }
 
-    /// Gets the integer value stored in the object.
+    /// \return The integer value stored by the object.
```

Comment:
Restore the one-liner description for the function? That's separate from `\return`. Same below.

(That's why I didn't write a `\return` in the first place, because it seemed redundant. But this seems strange as is.)

Reply:
I went back and forth on this a bit, and the thing is that specifying both is duplicative/redundant. However, specifying the `\return` statement solves both the "what this is" (a getter) and "what it does" (gets values) functionality.

Reply:
Mind updating the Style guide section in `CONTRIBUTING.md` to explain this for functions where the summary docstring and the `\return` would be essentially the same?